### PR TITLE
Adding deployment type to beta provider.

### PR DESCRIPTION
**Pull request by unnatinadupalli**
Adding deployment type to beta provider.

```release-note:enhancement
parallelstore: added `deployment_type` to `google_parallelstore_instance`
```

### CODE REVIEW

File: mmv1/templates/terraform/examples/parallelstore_instance_baic_beta.tf.tmpl [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  }
+  provider = google-beta
+  depends_on = [google_service_networking_connection.default]
+}
+
+resource "google_compute_network" "network" {
+  name                    = "{{index $.Vars "network_name"}}"
+  auto_create_subnetworks = true
+  mtu = 8896
+  provider = google-beta
```
Comment (c2thorn) [inline mmv1/templates/terraform/examples/parallelstore_instance_baic_beta.tf.tmpl:21]: Standard practice is to have the `provider = google-beta` line at the top of the resource (above the `name`) line. Do you mind making this change for all resources added in this PR?

**Review by c2thorn — CHANGES_REQUESTED** [review]

**Comment by c2thorn** [comment]
@unnatinadupalli I'd recommend following the initial steps here to get automatic build runs for any future pull requests: go/terraform-contribution-guide#before-you-begin

**Review by c2thorn — APPROVED** [review]

**Review by c2thorn — CHANGES_REQUESTED** [review]
```
Error: google/services/parallelstore/resource_parallelstore_instance_test.go:155:1: expected declaration, found '}'
```

**Comment by unnatinadupalli** [comment]
> ```
> Error: google/services/parallelstore/resource_parallelstore_instance_test.go:155:1: expected declaration, found '}'
> ```

Done

**Review by c2thorn — APPROVED** [review]

**Pull request merged**